libs/tokens/src/hooks/tokens/unsupported/useIsUnsupportedToken.ts

```diff
@@ -1,20 +1,16 @@
 import { useUnsupportedTokens } from './useUnsupportedTokens'
 import { useCallback } from 'react'
 
-export function useIsUnsupportedToken() {
+type NullishAddress = string | null | undefined
```

Comment:
Repeated type. What about move it to the token app types?

Reply:
Don't you mind if I do it in the next iteration?

Reply:
Sure, no problem